lib/class-wp-theme-json-data-gutenberg.php

```diff
@@ -38,7 +38,7 @@ class WP_Theme_JSON_Data_Gutenberg {
 	 * @param array  $data   Array following the theme.json specification.
 	 * @param string $origin The origin of the data: default, theme, user.
 	 */
-	public function __construct( $data = array(), $origin = 'theme' ) {
+	public function __construct( $data = array( 'version' => WP_Theme_JSON_Gutenberg::LATEST_SCHEMA ), $origin = 'theme' ) {
```

Comment:
I looked for all the places that define a default theme.json array and added the version to them. Without the version, they run all the migrations which would trigger adding the `'defaultFontSizes' => 'merge'` behavior.